### DISCUSSION TITLE
How to get the RF_output?

### DISCUSSION THREAD

**Discussion by francescomosconi** [SeisSol/SeisSol#882] [Q&A]
Hi, 

I ran the tpv104 example. 
Should it print a file with the rupture front output?
The parameters file is set as follow:
`RF_output_on = 1 OutputPointType = 5`

What the output file should be?

Thanks, 
Francesco

**Comment by Thomas-Ulrich** [comment]
Hi,
`RF_output_on = 1` is deprecated.
The RF variable can be read in the fault output.
